src/caretogether-pwa/src/Notes/NoteCard.tsx

```diff
@@ -32,6 +34,23 @@ export function NoteCard({ familyId, note }: NoteCardProps) {
 
   const permissions = useFamilyIdPermissions(familyId);
 
+  const userId = useLoadable(accountInfoState)?.userId;
+
+  const isOwnNote = note?.authorId === userId;
+
+  const canEditOwnNote =
```

Comment:
Just a comment for future consideration as we design for V2: the backend should tell the client which permissions the client has for each object (basically _HATEOAS_ from REST -- or whatever the equivalent pattern is when we implement GraphQL?) and the UI can then just render controls for whichever actions are allowed by the server. Otherwise we have to keep implementing this business logic on the client and server, which is basically why the current client code is so messy and difficult to work with.